includes/classes/order.php

```diff
@@ -161,7 +161,8 @@ function query($order_id) {
                             'country' => $order->fields['delivery_country'],
                             'format_id' => $order->fields['delivery_address_format_id']);
 
-    if (empty($this->delivery['name']) && empty($this->delivery['street_address'])) {
+    if (($order->fields['shipping_module_code'] == 'storepickup') || 
+        empty($this->delivery['name']) && empty($this->delivery['street_address'])) {
```

Comment:
Is this nesting of parentheses correct?  Shouldn't the extra opening `(` be moved from 164 to beginning of 165?

Reply:
No - and binds more tightly than or.  But I can add parens if you find this confusing to read.

Reply:
So then the first extra parentheses is redundant, right?

Reply:
You're right.

Reply:
Yes, but I think it's a bit easier to read how it is now.

Reply:
Fair. I'm always trying to avoid unnecessary visual debt, including unnecessary parentheses.